source/atlas-cli-deploy-docker.txt

```diff
@@ -0,0 +1,148 @@
+.. _atlas-cli-deploy-docker:
+
+=========================================
+Create a Local Atlas Deployment in Docker
+=========================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial shows you how to use the ``atlas deployments`` command to 
+create a local |service| deployment in Docker. In this tutorial, we 
+will deploy a single-node replica set in Docker.
+
+.. include:: /includes/fact-atlas-deployments-preview.rst
+
+.. _atlas-cli-deploy-local-reqs:
+
+Complete the Prerequisites
+--------------------------
+
```

Comment:
The only pre-requisite is to have Docker installed. None of these are needed for working with Docker.